openfpga_flow/tasks/skywater_openfpga_task

```diff
@@ -0,0 +1 @@
+/home/apond/sofa/SCRIPT/skywater_openfpga_task
```

Comment:
We do not put skywater task run in OpenFPGA repo. They are covered in the SOFA repository.
I see you have added new benchmarks. Can you create dedicated task-run for the picorv benchmarks?
The micro-benchmark regression test is a proper place to add these test cases.

https://github.com/lnis-uofu/OpenFPGA/blob/223e06d23c070b4d704a4e120b53e85d738cd162/openfpga_flow/regression_test_scripts/micro_benchmark_reg_test.sh#L19

Reply:
Yes, I thought I deleted the skywater_openfpga_task symlink; I'll delete it.
And yes, I will create a task-run for the picorv.